### ISSUE TITLE
Parent selectors don't get parsed

### ISSUE THREAD

**Issue by tonyganch** [tonyganch/gonzales-pe#100]
Line 4 throws an error:

``` scss
.layout-1 {
  @include breakpoint($breakpoint-large) {
    .section-header {
      &.img {
        .content {
          @include span(first 15 of 24);
          // ensure text doesn't come too close to the image
          @if supports-browser('ie7') {
            .ie7 & {
              @include span(first 14 of 24); // Less the post 1
            }
          }
          @include post(1);
          // leave a 3 column gap
        }

      }
    }
  }
}
```

Line 2 throws an error:

``` scss
.foo {
  &:after {
    content: 'foo';
  }
}
```

Moved from #96.

/cc @bgriffith @cgtm

**Comment by cgtm** [comment]
Just to confirm, I mentioned these problems on SCSS, not SASS. Are you putting them under [SASS] as verified as a problem there too?

**Comment by tonyganch** [comment]
I've added `sass` label to remember to check the same issue there too, though I suppose Sass should be fine.

**Comment by cgtm** [comment]
Cool! Just wanted to check.

**Comment by tonyganch** [comment]
Should be fixed in `3.0.1`. Please let me know if the issue's still there.
